Makefile

```diff
@@ -6,6 +6,15 @@
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
+dev:
+	go get github.com/axw/gocov/gocov
+	go get github.com/golang/lint/golint
+	go get github.com/mattn/goveralls
```

Comment:
I think line 12 is only needed by travis - this is for publishing coverage reports to coveralls.